pkg/byteconv/byteconv.go

```diff
@@ -20,10 +21,10 @@ func ParseBool(b []byte) (bool, error) {
 	return strconv.ParseBool(UnsafeString(b))
 }
 
-func ParseIP(b []byte) (net.IP, error) {
-	ip := net.ParseIP(UnsafeString(b))
-	if ip == nil {
-		return nil, fmt.Errorf("net.ParseIP: parsing %q: syntax error", b)
+func ParseIP(b []byte) (netaddr.IP, error) {
+	ip, err := netaddr.ParseIP(UnsafeString(b))
+	if err != nil {
+		return netaddr.IP{}, fmt.Errorf("net.ParseIP: parsing %q: %w", b, err)
 	}
 	return ip, nil
```

Comment:
Errors from `netaddr.ParseIP` are already prefixed with "ParseIP" and include the argument.
```suggestion
	return netaddr.ParseIP(UnsafeString(b))
```